main.go

```diff
@@ -37,6 +37,9 @@ func main() {
 	servercfg.SetVersion(version)
 	fmt.Println(models.RetrieveLogo()) // print the logo
 	initialize()                       // initial db and acls
+	logic.SetIpPool()
+	defer logic.ClearIpPool()
+	defer logic.WriteMetricsCacheToDB()
```

Comment:
Change it back in new commit. 

Regarding Check-in and metric data, every time when updating check-in or metrics, it writes the data to DB, overwriting the old data.  The UI only load the latest data.  From performance perspective,  writing to memory is better than writing to DB.

In the 1000 hosts scale test,  DB write is a not a primary blocker issue.  I changed it back to avoid inconsistent issue.